CMakeLists.txt

```diff
@@ -56,6 +56,14 @@ option(USE_ROCKSDB "Use RocksDB" ON)
 option(USE_SNPE "Use Qualcomm's SNPE library" OFF)
 option(USE_THREADS "Use Threads" ON)
 option(USE_ZMQ "Use ZMQ" OFF)
+if(MSVC)
+  if(BUILD_BUILD_SHARED_LIBS)
```

Comment:
Shouldn't the variable be 'BUILD_SHARED_LIBS'?

Reply:
@aralph You are right! Thanks for pointing out. I must'nt have had enough coffee when I wrote this. When I get back to caffe2 on Windows I will update the PR.